src/mitol/payment_gateway/api.py

```diff
@@ -252,3 +286,21 @@ def prepare_checkout(
             "url": settings.MITOL_PAYMENT_GATEWAY_CYBERSOURCE_SECURE_ACCEPTANCE_URL,
             "method": "POST",
         }
+
+    def perform_processor_response_validation(self, request):
+        """
+        CyberSource will send back a payload signed in the same manner that the
+        original transaction was sent, so we just reuse that process here, with
+        some logic to pull the proper request data out depending on HTTP verb.
+        """
+        if request.method == "GET":
+            passed_payload = request.query_params
+        else:
+            passed_payload = request.data
+
+        signature = self._generate_cybersource_sa_signature(passed_payload)
+
+        if passed_payload["signature"] == signature:
+            return True
+        else:
+            return False
```

Comment:
Can be simplified to:
```suggestion
        return passed_payload["signature"] == signature
```